rawkv/rawkv.go

```diff
@@ -185,33 +197,64 @@ func NewClient(ctx context.Context, pdAddrs []string, security config.Security,
 	return NewClientWithOpts(ctx, pdAddrs, WithSecurity(security), WithPDOptions(opts...))
 }
 
+func getKeyspaceID(pdCli pd.Client, keyspaceName string) (uint32, error) {
```

Comment:
It seems to have been implemented in txnkv/client.go ?

Reply:
Removed